test/EmailAddressTest.php

```diff
@@ -286,7 +286,13 @@ public function testComplexLocalValid()
             '/Bob.Jones@domain.com',
             '#Bob.Jones@domain.com',
             'Bob.Jones?@domain.com',
-            'Bob~Jones@domain.com'
+            'Bob~Jones@domain.com',
+            // RFC 5321 does mention a limit of 64 for the username,
+            // but it also states "To the maximum extent possible,
+            // implementation techniques that impose no limits on the
+            // length of these objects should be used.".
+            // http://tools.ietf.org/html/rfc5321#section-4.5.3.1
+            'line length 911' => str_repeat('x', 900).'@domain.com'
```

Comment:
Do this test with 320 and 321 lengths instead. Reflect better the previous length barrier